ui/raidboss/data/06-ew/raid/p2s.txt

```diff
@@ -0,0 +1,118 @@
+105.1 "Ominous Bubbling" sync / 1[56]:[^:]*:Hippokampos:682A:/
+115.9 "Shockwave" sync / 1[56]:[^:]*:Hippokampos:682F:/
+119.7 "Ominous Bubbling Groups" sync / 1[56]:[^:]*:Hippokampos:682B:/
+
+# Predatory Avarice and Channeling Flow
+128.1 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6836:/
+134.5 "Predatory Avarice" sync / 1[56]:[^:]*:Hippokampos:6827:/
+152.2 "Winged Cataract/Spoken Cataract" sync / 1[56]:[^:]*:Hippokampos:(6811|6812|6813|6814|6815|6816|6817|6818):/
+158.5 "Hard Water/Deadly Current" sync / 1[56]:[^:]*:Hippokampos:(6828|6829):/
+171.9 "Channeling Flow" sync / 1[56]:[^:]*:Hippokampos:681E:/
+187.2 "Great Typhoon" sync / 1[56]:[^:]*:Hippokampos:(6820|6822):/
+187.7 "Crash" #sync / 1[56]:[^:]*:Hippokampos:6821:/
+202.2 "Doubled Impact" sync / 1[56]:[^:]*:Hippokampos:6832:/
+212.3 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:6833:/
+
+# Kampeos Harma
+221.5 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6836:/
+228.9 "Sewage Deluge" sync / 1[56]:[^:]*:Hippokampos:6810:/
+246.4 "Shockwave" sync / 1[56]:[^:]*:Hippokampos:682F:/
+259.3 "Kampeos Harma" sync / 1[56]:[^:]*:Hippokampos:6823:/
+259.7 "Kampeos Harma 1" #sync / 1[56]:[^:]*:Hippokampos:(6825|6824):/
+261.2 "Kampeos Harma 2 (head)" #sync / 1[56]:[^:]*:Hippokampos:6825:/
+261.5 "Kampeos Harma 2 (body)" #sync / 1[56]:[^:]*:Hippokampos:6824:/
+262.8 "Kampeos Harma 3 (head)" #sync / 1[56]:[^:]*:Hippokampos:6825:/
+263.3 "Kampeos Harma 3 (body)" #sync / 1[56]:[^:]*:Hippokampos:6824:/
+264.5 "Kampeos Harma 4 (head)" sync / 1[56]:[^:]*:Hippokampos:6826:/
+265.2 "Kampeos Harma 4 (body)" sync / 1[56]:[^:]*:Hippokampos:6824:/
+266.8 "--sync--" sync / 1[56]:[^:]*:Hippokampos:682D:/
+269.9 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6836:/
+281.7 "Doubled Impact" sync / 1[56]:[^:]*:Hippokampos:6832:/
+290.8 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:6833:/
+
+# Timed Channeling Flow 1
+297.0 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6836:/
+304.4 "Channeling Overflow" sync / 1[56]:[^:]*:Hippokampos:6DC2:/
+311.6 "--sync--" sync / 1[56]:[^:]*:Hippokampos:6837:/
+318.7 "Tainted Flood" sync / 1[56]:[^:]*:Hippokampos:6838:/
+319.7 "Great Typhoon (short)" sync / 1[56]:[^:]*:Hippokampos:(6820|6822):/
+320.2 "Crash" #sync / 1[56]:[^:]*:Hippokampos:6821:/
+327.0 "--sync--" sync / 1[56]:[^:]*:Hippokampos:6837:/
+334.1 "Tainted Flood" sync / 1[56]:[^:]*:Hippokampos:6838:/
+334.7 "Great Typhoon (long)" sync / 1[56]:[^:]*:Hippokampos:(6820|6822):/
+335.2 "Crash" #sync / 1[56]:[^:]*:Hippokampos:6821:/
+345.2 "Spoken Cataract/Winged Cataract" sync / 1[56]:[^:]*:Hippokampos:(6811|6812|6813|6814|6815|6816|6817|6818):/
+
+# Dissociation and Sewage Eruption 1
+357.9 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6836:/
+364.4 "Predatory Avarice" sync / 1[56]:[^:]*:Hippokampos:6827:/
+370.8 "Dissociation" sync / 1[56]:[^:]*:Hippokampos:682C:/
+375.9 "--sync--" sync / 1[56]:[^:]*:Hippokampos:682D:/
+388.5 "Hard Water/Deadly Current" sync / 1[56]:[^:]*:Hippokampos:(6828|6829):/

```

Comment:
Can you separate these out into two different lines? Mostly I think of A/B as meaning A or B, and this is always A and B.

Reply:
I also noticed that one of these combos happens within 0.5 sec of a winged/spoken cataract so I commented the cataract sync that occurs after.